services/forms-api/openapi.yml

```diff
@@ -0,0 +1,464 @@
+openapi: '3.0.3'
+info:
+  title: Forms
+  description: Api for forms made with form-builder.
+  contact:
+    name: GitHub Repo
+    url: https://github.com/helsingborg-stad/helsingborg-io-sls-api
+  version: '1.0'
+  license:
+    name: MIT
+    url: 'https://github.com/helsingborg-stad/helsingborg-io-sls-api#license'
+servers:
+  - url: https://dev.api.helsingborg.io
+  - url: https://release.api.helsingborg.io
+  - url: https://api.helsingborg.io
+
+paths:
+  /forms:
+    get:
+      tags:
+        - Forms
+      summary: Get all forms.
+      description: This endpoint returns a list of all forms created in form-builder tool.
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/FormListResponse'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+
+    post:
+      tags:
+        - Forms
+      summary: Creates form.
+      description: This endpoint creates a new form-builder form entry in the database.
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Form'
+      responses:
+        '201':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+
+  /forms/{formId}:
+    get:
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Get form.
+      description: This endpoint returns a form-builder form by its unique id.
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+
+    put:
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Updates a form.
+      description: This endpoint updates a current form.
```

Comment:
Might add that if `formId` does not point to an existing form a new one is actually created (possibly a bug in forms-api but that's what it does).

Reply:
fixed